### DISCUSSION TITLE
Showing a few changes I made

### DISCUSSION THREAD

**Discussion by thisisshadow** [Sketchware-Pro/Sketchware-Pro#1158] [Show and tell]
I want to contribute to Sketchware Pro and will generally be interested in design-related things, that is, I will work on material redesign and correct the uncorrected parts.

The change I mentioned looks like the one below, it was good for me, I want to ask how it was for you, do you think it is good?

- Light
![IMG_20240605_221706](https://github.com/Sketchware-Pro/Sketchware-Pro/assets/116355867/211a1ecb-6209-450b-9d4a-2f85ed0b7d52)

- Dark
![IMG_20240605_222840](https://github.com/Sketchware-Pro/Sketchware-Pro/assets/116355867/d68052c9-36bd-44f4-b51a-fc9ebcbd83b4)

- A few changes I made in the Firebase Manager section
• Here I edited the change firebase config page
![IMG_20240605_223038](https://github.com/Sketchware-Pro/Sketchware-Pro/assets/116355867/a6ae8c29-3691-46de-ab01-c1a66dd913cd)

• And let me also point out that the firebase config page (kv.class) was of class type and I deleted it from the jar file and converted this class file to java and added it to the source codes of the application. Because otherwise, when there were two classes, these classes would overlap.

- I also renewed the Export Project activity and it looked better than before.
![IMG_20240605_223530](https://github.com/Sketchware-Pro/Sketchware-Pro/assets/116355867/3cfaf273-70f3-4796-a7b5-451161c03bbd)

- Code previews follow when the phone is put in dark mode. (some of them are like this for now)
![IMG_20240605_224010](https://github.com/Sketchware-Pro/Sketchware-Pro/assets/116355867/5559f392-478c-4570-b7c9-02b3d1e443a4)
![IMG_20240605_224023](https://github.com/Sketchware-Pro/Sketchware-Pro/assets/116355867/4b14ae49-8a0f-433e-b395-b6f5ce44ed35)

• I've done things like this for now, then I'll edit the pages that need to be converted to material design.

**Comment by RoyTheReal** [comment]
nice!

**Reply by thisisshadow** [reply]
Thanks

**Comment by RoyTheReal** [comment]
can you update all switches to material switches?

**Reply by thisisshadow** [reply]
Are they the switches within the project or the switches for the settings of sketchware?

**Reply by RoyTheReal** [reply]
> Are they the switches within the project or the switches for the settings of sketchware?

Update the switches within the settings for Sketchware

**Reply by thisisshadow** [reply]
> > Are they the switches within the project or the switches for the settings of sketchware?
> 
> Update the switches within the settings for Sketchware

I will try bro

**Reply by thisisshadow** [reply]
> > Are they the switches within the project or the switches for the settings of sketchware?
> 
> Update the switches within the settings for Sketchware

Dude, I redesigned pages you mentioned with material view.

**Reply by RoyTheReal** [reply]
> > > Are they the switches within the project or the switches for the settings of sketchware?
> > 
> > 
> > Update the switches within the settings for Sketchware
> 
> Dude, I redesigned pages you mentioned with material view.

Nice man! Thanks. Can you now upgrade to compiling dialog to be more professional and be material 3 styled? 
![Screenshot_20240609_141150_Sketchware Pro](https://github.com/Sketchware-Pro/Sketchware-Pro/assets/41343639/b03980bf-5f32-4710-b103-83980ee63959)
↑ Update this to be more professional and don't make it ask the questions. Just make it a Material 3 dialog or something

**Reply by thisisshadow** [reply]
> > > > Are they the switches within the project or the switches for the settings of sketchware?
> > > 
> > > 
> > > Update the switches within the settings for Sketchware
> > 
> > 
> > Dude, I redesigned pages you mentioned with material view.
> 
> Nice man! Thanks. Can you now upgrade to compiling dialog to be more professional and be material 3 styled? ![Screenshot_20240609_141150_Sketchware Pro](https://private-user-images.githubusercontent.com/41343639/337998008-b03980bf-5f32-4710-b103-83980ee63959.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc5NTc4ODksIm5iZiI6MTcxNzk1NzU4OSwicGF0aCI6Ii80MTM0MzYzOS8zMzc5OTgwMDgtYjAzOTgwYmYtNWYzMi00NzEwLWIxMDMtODM5ODBlZTYzOTU5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA5VDE4MjYyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU1MDFiZjdkNDM3NjE0N2M3NTM5ZDc5ZjViNjExZTQ5MGFmNTdlMWJhYjBkZGUyM2IwYWU3MjRjNjRmMGNkZWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zx1B1LxQe-1wNW6QvV7mgHv7fsyxB2vnnUt2qCIAe8s) ↑ Update this to be more professional and don't make it ask the questions. Just make it a Material 3 dialog or something

Bro, I actually have an idea for this project compilation box, but I'm not sure if it will be good or not.

**Reply by RoyTheReal** [reply]
You should do it. Just test a bunch of different designs

**Reply by thisisshadow** [reply]
> You should do it. Just test a bunch of different designs

I have an idea in my mind like creating a foreground service and not making this dialogue mandatory, maybe it won't be good, but I'll test it anyway.

**Reply by RoyTheReal** [reply]
> > You should do it. Just test a bunch of different designs
> 
> I have an idea in my mind like creating a foreground service and not making this dialogue mandatory, maybe it won't be good, but I'll test it anyway.

I have faith in you to make it look good 💪

**Reply by thisisshadow** [reply]
> > > You should do it. Just test a bunch of different designs
> > 
> > 
> > I have an idea in my mind like creating a foreground service and not making this dialogue mandatory, maybe it won't be good, but I'll test it anyway.
> 
> I have faith in you to make it look good 💪

But maybe I can add an option to the mode settings such as use the new compiler mode. When it is disabled, it will use the mode I added in the foreground service while it is active with this dialog.

**Comment by PranavPurwar** [comment]
I think the background color of view properties should be a bit light and export projects screen looks a bit weird. The rest of them look really good though. 👍

**Reply by thisisshadow** [reply]
Thank you bro, I will implement the suggestion you mentioned in the view properties section, but why does the export project page seem strange to you? 😅

**Reply by PranavPurwar** [reply]
The cardview doesn't look like its supposed to be there idk

**Comment by ilyassesalama** [comment]
Your changes are very nice. Looking forward to seeing your pull request. Thanks for your contribution.

**Reply by thisisshadow** [reply]
Thank you buddy, there is already a pull request waiting for approval.

**Comment by treadom** [comment]
Can you give me this app please
